dataprofiler/data_readers/json_data.py

```diff
@@ -419,4 +434,4 @@ def reload(self, input_file_path=None, data=None, options=None):
         """
         self._selected_keys = None
         super(JSONData, self).reload(input_file_path, data, options)
-        self.__init__(self.input_file_path, data, options)
+        JSONData.__init__(self, self.input_file_path, data, options)
```

Comment:
was mypy complaining here?

Reply:
Yes, mypy complained about calling self.__init__ but I think this change should do the same thing.

Reply:
Could that be problematic for anything that inherits it? For instance if avro didn't have a reload, it would only call JSONData reload which I think could cause a problem.

Reply:
+1